fontir/src/ir.rs

```diff
@@ -719,33 +718,86 @@ impl GlobalMetrics {
 #[derive(Default, Debug, Clone, PartialEq, Eq)]
 pub struct GlobalMetricsInstance {
     pub pos: NormalizedLocation,
-    pub ascender: OrderedFloat<f32>,
-    pub descender: OrderedFloat<f32>,
-    pub caret_slope_rise: OrderedFloat<f32>,
-    pub caret_slope_run: OrderedFloat<f32>,
-    pub caret_offset: OrderedFloat<f32>,
-    pub cap_height: OrderedFloat<f32>,
-    pub x_height: OrderedFloat<f32>,
-    pub os2_typo_ascender: OrderedFloat<f32>,
-    pub os2_typo_descender: OrderedFloat<f32>,
-    pub os2_typo_line_gap: OrderedFloat<f32>,
-    pub os2_win_ascent: OrderedFloat<f32>,
-    pub os2_win_descent: OrderedFloat<f32>,
-    pub hhea_ascender: OrderedFloat<f32>,
-    pub hhea_descender: OrderedFloat<f32>,
-    pub hhea_line_gap: OrderedFloat<f32>,
-    pub strikeout_position: OrderedFloat<f32>,
-    pub strikeout_size: OrderedFloat<f32>,
-    pub subscript_x_offset: OrderedFloat<f32>,
-    pub subscript_x_size: OrderedFloat<f32>,
-    pub subscript_y_offset: OrderedFloat<f32>,
-    pub subscript_y_size: OrderedFloat<f32>,
-    pub superscript_x_offset: OrderedFloat<f32>,
-    pub superscript_x_size: OrderedFloat<f32>,
-    pub superscript_y_offset: OrderedFloat<f32>,
-    pub superscript_y_size: OrderedFloat<f32>,
-    pub underline_thickness: OrderedFloat<f32>,
-    pub underline_position: OrderedFloat<f32>,
+    pub ascender: OrderedFloat<f64>,
+    pub descender: OrderedFloat<f64>,
+    pub caret_slope_rise: OrderedFloat<f64>,
+    pub caret_slope_run: OrderedFloat<f64>,
+    pub caret_offset: OrderedFloat<f64>,
+    pub cap_height: OrderedFloat<f64>,
+    pub x_height: OrderedFloat<f64>,
+    pub os2_typo_ascender: OrderedFloat<f64>,
+    pub os2_typo_descender: OrderedFloat<f64>,
+    pub os2_typo_line_gap: OrderedFloat<f64>,
+    pub os2_win_ascent: OrderedFloat<f64>,
+    pub os2_win_descent: OrderedFloat<f64>,
+    pub hhea_ascender: OrderedFloat<f64>,
+    pub hhea_descender: OrderedFloat<f64>,
+    pub hhea_line_gap: OrderedFloat<f64>,
+    pub strikeout_position: OrderedFloat<f64>,
+    pub strikeout_size: OrderedFloat<f64>,
+    pub subscript_x_offset: OrderedFloat<f64>,
+    pub subscript_x_size: OrderedFloat<f64>,
+    pub subscript_y_offset: OrderedFloat<f64>,
+    pub subscript_y_size: OrderedFloat<f64>,
+    pub superscript_x_offset: OrderedFloat<f64>,
+    pub superscript_x_size: OrderedFloat<f64>,
+    pub superscript_y_offset: OrderedFloat<f64>,
+    pub superscript_y_size: OrderedFloat<f64>,
+    pub underline_thickness: OrderedFloat<f64>,
+    pub underline_position: OrderedFloat<f64>,
+}
+
+// It's not *that* fun to write long decimals in unit tests
+#[doc(hidden)]
+pub mod test_helpers {
+    use ordered_float::OrderedFloat;
+
+    use super::GlobalMetricsInstance;
+
+    pub trait Round2 {
```

Comment:
staring at this it isn't really evident how it's helpful. Maybe a little bit of exposition?

Reply:
Ty, will add